docs/analyzer.md

```diff
@@ -16,10 +16,11 @@ PM> Install-Package FakeItEasy.Analyzer
 
 ## Diagnostics
 
-The analyzer currently provides two diagnostics:
+The analyzer currently provides the following diagnostics:
 
-| Id             | Summary                               | Description                                                                                                                                           |
-|----------------|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
-| FakeItEasy0001 | Unused call specification             | Triggered when you specify a call but don't configure or assert it, making it a no-op.                                                                |
-| FakeItEasy0002 | Non-virtual member configuration      | Triggered when you try to configure a non-virtual member, which cannot be faked.                                                                      |
-| FakeItEasy0003 | Argument constraint outside call spec | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
+| Id             | Summary                                  | Description                                                                                                                                           |
+|----------------|------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FakeItEasy0001 | Unused call specification                | Triggered when you specify a call but don't configure or assert it, making it a no-op.                                                                |
+| FakeItEasy0002 | Non-virtual member configuration         | Triggered when you try to configure a non-virtual member, which cannot be faked.                                                                      |
+| FakeItEasy0003 | Argument constraint outside call spec    | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
+| FakeItEasy0004 | Argument constraint nullability mismatch | Triggered when you use a non-nullable [argument constraint](argument-constraints.md) for a nullable parameter. Calls where the argument is null won't be matched. If this is intentional, consider using `A<T?>.That.IsNotNull()` instead. If it's not, make the argument constraint nullable (`A<T?>`). |

```

Comment:
After a quick search, I did no better than you. Let's live with it for now.
OH! OH! I thought of a way:

```html
<table>
<tr>
…
```

😉